lib/mj-librsvg.js

```diff
@@ -0,0 +1,26 @@
+var rsvg = require('librsvg').Rsvg;
+var Readable = require('stream').Readable;
+
+exports.svg2png = function(svg, result, scale, width, height, callback, check) {
+  var s = new Readable();
+  var svgRenderer = new rsvg();
```

Comment:
You can avoid the stream altogether here since you already have the data to render in `svg`, so simply:

```js
var svgRender = new rsvg(svg);
```